### DISCUSSION TITLE
Prisma Client Initialization Error

### DISCUSSION THREAD

**Discussion by sireeshdevaraj** [prisma/prisma#14224] [Q&A] [answered]
### Bug description

I'm getting this error 
```
Invalid `_api_auth_prismaInit__WEBPACK_IMPORTED_MODULE_2__.prisma.user.findMany()` invocation at `const allUsers = await _api_auth_prismaInit__WEBPACK_IMPORTED_MODULE_2__.prisma.user.findMany()`
var _api_auth_prismaInit__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! ./api/auth/prismaInit */ "./pages/api/auth/prismaInit.js");
```

Below is the prismaInit.js file
```js
import { PrismaClient } from '@prisma/client';

export const prisma =
  global.prisma ||
  new PrismaClient({
    log: ['query'],
  })

if (process.env.NODE_ENV !== 'production') global.prisma = prisma
```

### Environment & setup

- OS: Windows
- Database: mysql
- Node.js version: v16.16.0

### Prisma Version

```
prisma                  : 4.0.0
@prisma/client          : 3.3.0
Current platform        : windows
```

**Comment by janpio** [comment, marked as answer]
Please start by updating your `@prisma/client` version to 4.0.0 as well and try again. Combinations of new CLI and pretty old Client often do not work.

**Reply by sireeshdevaraj** [reply]
Sorry, it was a issue with my database. It's solved. Got the issue because of the password encoding being done wrongly because of the '@' and '$' in the database URL

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
